### PR TITLE
Support for kandinsky 2.2 model

### PR DESCRIPTION
**Pull request by kodxana**
Added model_version to select if user wants to use 2.2 or 2.1 model
If not set will use 2.2 model.

IMG2IMG should work with 2.1 for 2.2 needs be implemented.

### CODE REVIEW

File: src/rp_handler.py [hunk 1]
```diff
@@ -2,37 +2,39 @@
...
+pipe_prior_2_1 = DiffusionPipeline.from_pretrained(
     "kandinsky-community/kandinsky-2-1-prior", torch_dtype=torch.float16).to("cuda")
-
-t2i_pipe = DiffusionPipeline.from_pretrained(
+t2i_pipe_2_1 = DiffusionPipeline.from_pretrained(
     "kandinsky-community/kandinsky-2-1", torch_dtype=torch.float16).to("cuda")
-t2i_pipe.enable_xformers_memory_efficient_attention()
+t2i_pipe_2_1.enable_xformers_memory_efficient_attention()
+
+# Kandinsky 2.2 pipelines
```
Comment (justinmerrell) [inline src/rp_handler.py:24]: Do these lines to load the model into memory fo inference?

Comment (kodxana) [reply src/rp_handler.py:24]: This line is to run the model based on difusser implementation with xformers optimisation.

File: src/rp_schemas.py [hunk 2]
```diff
@@ -3,10 +3,16 @@
...
 INPUT_SCHEMA = {
+    
     'prompt': {
         'type': str,
         'required': True,
     },
+    'model_version': {
+    'type': str,
+    'required': False,
+    'default': '2.2'
```
Comment (justinmerrell) [inline src/rp_schemas.py:14]: Our current endpoint is 2.1 correct? 

We should probably leave that as the default, and then may you add this new input option to the readme?

Comment (kodxana) [reply src/rp_schemas.py]: Edited to use 2.1 and also added to docs.

**Pull request merged**